Packs/Barracuda(BRBL)/Integrations/BarracudaReputationBlockListBRBL/BarracudaReputationBlockListBRBL.yml

```diff
@@ -0,0 +1,51 @@
+commonfields:
+  id: Barracuda Reputation Block List (BRBL)
+  version: -1
+vcShouldKeepItemLegacyProdMachine: false
```

Comment:
the field vcshouldkeepitemlegacyprodmachine was not defined in the scheme
To fix the problem, try running `demisto-sdk format -i Packs/Barracuda(BRBL)/Integrations/BarracudaReputationBlockListBRBL/BarracudaReputationBlockListBRBL.yml `